larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.h

```diff
@@ -194,6 +223,8 @@ class ThreeDOpeningAngleFeatureTool : public PfoCharacterisationFeatureTool
         pandora::CartesianPointVector &pointVectorStart, pandora::CartesianPointVector &pointVectorEnd);
 
     float OpeningAngle(const pandora::CartesianVector &principal, const pandora::CartesianVector &secondary, const pandora::CartesianVector &eigenValues) const;
+
```

Comment:
Remove whitespace

Reply:
Not my code block. This used to be Nick's code. Unsure as to what I should do with these. Should i just delete his code block?